bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -52,9 +52,6 @@
 data-try-now="{{ _('Try it now') }}"
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
I've opened a bug for this issue. https://bugzilla.mozilla.org/show_bug.cgi?id=1188852